pkg/e2e/operators/rbac.go

```diff
@@ -50,6 +52,9 @@ var _ = ginkgo.Describe(rbacOperatorBlocking, func() {
 var _ = ginkgo.Describe(subjectPermissionsTestName, func() {
 	h := helper.New()
 	checkSubjectPermissions(h, "dedicated-admins")
+	checkSubjectPermissions(h, "sre-admins")
+	daCreateSubjectpernission(h)
```

Comment:
`daCreateSubjectpernission` and `checkSubjectpernission` aren't reused. I would put their ginkgo Context blocks back under the Describe block, there's no benefit to breaking these out into separate functions.